river/tree/hoeffding_tree_classifier.py

```diff
@@ -40,8 +40,34 @@ class HoeffdingTreeClassifier(BaseHoeffdingTree, base.Classifier):
     nb_threshold
         Number of instances a leaf should observe before allowing Naive Bayes.
     nominal_attributes
-        List of Nominal attributes identifiers. If empty, then assume that all numeric attributes
-        should be treated as continuous.
+        List of Nominal attributes identifiers. If empty, then assume that all numeric
```

Comment:
Once more.

Reply:
Updated in commit [d247eb7](https://github.com/creme-ml/creme/pull/379/commits/d247eb7b873f17442e35e1b1df8054ad05611a4b).